docs/autodoc_submodules.py

```diff
@@ -51,23 +66,49 @@ def op_autodoc(out_filename):
         if module in exclude_ops_members:
             excluded = exclude_ops_members[module]
             s += "   :exclude-members: {}\n".format(", ".join(excluded))
-        s += "\n"
+        s += "\n\n"
     with open(out_filename, 'w') as f:
         f.write(s)
 
-def fn_autodoc(out_filename):
+def get_module_references(module_name, references):
+    """Generate section with references for given module"""
+    module_name = module_name[12:] # remove nvidia.dali prefix
+    result = ""
+    if module_name in references:
+        result += ".. seealso::\n"
+        for desc, url in references[module_name]:
+            result += f"   * `{desc} <./{url}>`_\n"
+    return result
+
+def get_operator_references(op_name, references):
```

Comment:
I think it is very similar to `get_module_references`, why do we need two separate functions for the same thing (I may have missing something obvious)?

Reply:
I started with a bit more customized text, but they are now virtually the same. Will remove one.

Reply:
done